<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Guitar Music Notes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_next_note_settings">
      <property name="title">
       <string>Next Note Selection</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_next_note_raido_buttons">
           <property name="styleSheet">
            <string notr="true">QGroupBox {border:0;}</string>
           </property>
           <property name="title">
            <string/>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QRadioButton" name="radioButton_timer">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Timer</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton_manually">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Next Note manually</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_timer_settings">
         <property name="font">
          <font>
           <stylestrategy>PreferAntialias</stylestrategy>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QGroupBox {border:0;}</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEdit_timer_seconds">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhDigitsOnly</set>
            </property>
            <property name="text">
             <string>4</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_timer_seconds">
            <property name="text">
             <string>seconds</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_follow_with_solution">
         <property name="text">
          <string>Show Solution</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label_selection_hint">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>Note: If no note is selected, all notes will be trained!</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_select_notes">
      <property name="title">
       <string>Select Notes</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="13" column="0">
        <widget class="QCheckBox" name="note_c">
         <property name="text">
          <string>c</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="note__g">
         <property name="text">
          <string>G</string>
         </property>
        </widget>
       </item>
       <item row="14" column="4">
        <widget class="QCheckBox" name="note_b__">
         <property name="text">
          <string>b ' '</string>
         </property>
        </widget>
       </item>
       <item row="17" column="0">
        <widget class="QCheckBox" name="note_dis">
         <property name="text">
          <string># d</string>
         </property>
        </widget>
       </item>
       <item row="16" column="4">
        <widget class="QCheckBox" name="note_c___">
         <property name="text">
          <string>c ' ' '</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QCheckBox" name="note_cis">
         <property name="text">
          <string># c</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="note__fis">
         <property name="text">
          <string># F</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QCheckBox" name="note__ais">
         <property name="text">
          <string># A</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QCheckBox" name="note__as">
         <property name="text">
          <string>â™­A</string>
         </property>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QCheckBox" name="note_d">
         <property name="text">
          <string>d</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QCheckBox" name="note__gis">
         <property name="text">
          <string># G</string>
         </property>
        </widget>
       </item>
       <item row="12" column="3">
        <widget class="QCheckBox" name="note_ais_">
         <property name="text">
          <string># a '</string>
         </property>
        </widget>
       </item>
       <item row="14" column="3">
        <widget class="QCheckBox" name="note_h_">
         <property name="text">
          <string>h '</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QCheckBox" name="note__h">
         <property name="text">
          <string>H</string>
         </property>
        </widget>
       </item>
       <item row="13" column="3">
        <widget class="QCheckBox" name="note_b_">
         <property name="text">
          <string>b '</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QCheckBox" name="note__a">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item row="15" column="3">
        <widget class="QCheckBox" name="note_c__">
         <property name="text">
          <string>c ' '</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="note__e">
         <property name="text">
          <string>E</string>
         </property>
        </widget>
       </item>
       <item row="10" column="4">
        <widget class="QCheckBox" name="note_gis__">
         <property name="text">
          <string># g ' '</string>
         </property>
        </widget>
       </item>
       <item row="15" column="4">
        <widget class="QCheckBox" name="note_h__">
         <property name="text">
          <string>h ' '</string>
         </property>
        </widget>
       </item>
       <item row="11" column="4">
        <widget class="QCheckBox" name="note_as__">
         <property name="text">
          <string>â™­a ' '</string>
         </property>
        </widget>
       </item>
       <item row="12" column="4">
        <widget class="QCheckBox" name="note_a__">
         <property name="text">
          <string>a ' '</string>
         </property>
        </widget>
       </item>
       <item row="13" column="4">
        <widget class="QCheckBox" name="note_ais__">
         <property name="text">
          <string># a ' '</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="note__ges">
         <property name="text">
          <string>â™­G</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="note__f">
         <property name="text">
          <string>F</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <widget class="QCheckBox" name="note__b">
         <property name="text">
          <string>B</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QCheckBox" name="note_des">
         <property name="text">
          <string>â™­d</string>
         </property>
        </widget>
       </item>
       <item row="7" column="4">
        <widget class="QCheckBox" name="note_fis__">
         <property name="text">
          <string># f ' '</string>
         </property>
        </widget>
       </item>
       <item row="6" column="4">
        <widget class="QCheckBox" name="note_f__">
         <property name="text">
          <string>f ' '</string>
         </property>
        </widget>
       </item>
       <item row="8" column="4">
        <widget class="QCheckBox" name="note_ges__">
         <property name="text">
          <string>â™­g ' '</string>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <widget class="QCheckBox" name="note_es__">
         <property name="text">
          <string>â™­e ' '</string>
         </property>
        </widget>
       </item>
       <item row="9" column="4">
        <widget class="QCheckBox" name="note_g__">
         <property name="text">
          <string>g ' '</string>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QCheckBox" name="note_e__">
         <property name="text">
          <string>e ' '</string>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QCheckBox" name="note_dis__">
         <property name="text">
          <string># d ' '</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QCheckBox" name="note_d__">
         <property name="text">
          <string>d ' '</string>
         </property>
        </widget>
       </item>
       <item row="11" column="3">
        <widget class="QCheckBox" name="note_a_">
         <property name="text">
          <string>a '</string>
         </property>
        </widget>
       </item>
       <item row="16" column="3">
        <widget class="QCheckBox" name="note_cis__">
         <property name="text">
          <string># c ' '</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QCheckBox" name="note_f">
         <property name="text">
          <string>f</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QCheckBox" name="note_gis">
         <property name="text">
          <string># g</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QCheckBox" name="note_fis">
         <property name="text">
          <string># f</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QCheckBox" name="note_g">
         <property name="text">
          <string>g</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QCheckBox" name="note_as">
         <property name="text">
          <string>â™­a</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QCheckBox" name="note_ges">
         <property name="text">
          <string>â™­g</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QCheckBox" name="note_e">
         <property name="text">
          <string>e</string>
         </property>
        </widget>
       </item>
       <item row="11" column="2">
        <widget class="QCheckBox" name="note_ais">
         <property name="text">
          <string># a</string>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QCheckBox" name="note_a">
         <property name="text">
          <string>a</string>
         </property>
        </widget>
       </item>
       <item row="12" column="2">
        <widget class="QCheckBox" name="note_b">
         <property name="text">
          <string>b</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QCheckBox" name="note_es">
         <property name="text">
          <string>â™­e</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QCheckBox" name="note_e_">
         <property name="text">
          <string>e '</string>
         </property>
        </widget>
       </item>
       <item row="16" column="2">
        <widget class="QCheckBox" name="note_des_">
         <property name="text">
          <string>â™­d '</string>
         </property>
        </widget>
       </item>
       <item row="17" column="3">
        <widget class="QCheckBox" name="note_des__">
         <property name="text">
          <string>â™­d ' '</string>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QCheckBox" name="note_f_">
         <property name="text">
          <string>f '</string>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QCheckBox" name="note_fis_">
         <property name="text">
          <string># f '</string>
         </property>
        </widget>
       </item>
       <item row="14" column="2">
        <widget class="QCheckBox" name="note_c_">
         <property name="text">
          <string>c '</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QCheckBox" name="note_dis_">
         <property name="text">
          <string># d '</string>
         </property>
        </widget>
       </item>
       <item row="15" column="2">
        <widget class="QCheckBox" name="note_cis_">
         <property name="text">
          <string># c '</string>
         </property>
        </widget>
       </item>
       <item row="17" column="2">
        <widget class="QCheckBox" name="note_d_">
         <property name="text">
          <string>d '</string>
         </property>
        </widget>
       </item>
       <item row="13" column="2">
        <widget class="QCheckBox" name="note_h">
         <property name="text">
          <string>h</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QCheckBox" name="note_es_">
         <property name="text">
          <string>â™­e '</string>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <widget class="QCheckBox" name="note_ges_">
         <property name="text">
          <string>â™­g '</string>
         </property>
        </widget>
       </item>
       <item row="10" column="3">
        <widget class="QCheckBox" name="note_as_">
         <property name="text">
          <string>â™­a '</string>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="QCheckBox" name="note_g_">
         <property name="text">
          <string>g '</string>
         </property>
        </widget>
       </item>
       <item row="9" column="3">
        <widget class="QCheckBox" name="note_gis_">
         <property name="text">
          <string># g '</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>250</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_start_training">
        <property name="text">
         <string>Start Training</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Close"/>
   </widget>
   <widget class="QMenu" name="menuAbout_Qt">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout_Guitar_Music_Notes"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuAbout_Qt"/>
  </widget>
  <action name="action_Close">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionAbout_Guitar_Music_Notes">
   <property name="text">
    <string>About &amp;Guitar Music Notes</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>radioButton_timer</tabstop>
  <tabstop>radioButton_manually</tabstop>
  <tabstop>lineEdit_timer_seconds</tabstop>
  <tabstop>note__e</tabstop>
  <tabstop>note__f</tabstop>
  <tabstop>note__fis</tabstop>
  <tabstop>note__ges</tabstop>
  <tabstop>note__g</tabstop>
  <tabstop>note__gis</tabstop>
  <tabstop>note__as</tabstop>
  <tabstop>note__a</tabstop>
  <tabstop>note__ais</tabstop>
  <tabstop>note__b</tabstop>
  <tabstop>note__h</tabstop>
  <tabstop>note_c</tabstop>
  <tabstop>note_cis</tabstop>
  <tabstop>note_des</tabstop>
  <tabstop>note_d</tabstop>
  <tabstop>note_dis</tabstop>
  <tabstop>note_es</tabstop>
  <tabstop>note_e</tabstop>
  <tabstop>note_f</tabstop>
  <tabstop>note_fis</tabstop>
  <tabstop>note_ges</tabstop>
  <tabstop>note_g</tabstop>
  <tabstop>note_gis</tabstop>
  <tabstop>note_as</tabstop>
  <tabstop>note_a</tabstop>
  <tabstop>note_ais</tabstop>
  <tabstop>note_b</tabstop>
  <tabstop>note_h</tabstop>
  <tabstop>note_c_</tabstop>
  <tabstop>note_cis_</tabstop>
  <tabstop>note_des_</tabstop>
  <tabstop>note_d_</tabstop>
  <tabstop>note_dis_</tabstop>
  <tabstop>note_es_</tabstop>
  <tabstop>note_e_</tabstop>
  <tabstop>note_f_</tabstop>
  <tabstop>note_fis_</tabstop>
  <tabstop>note_ges_</tabstop>
  <tabstop>note_g_</tabstop>
  <tabstop>note_gis_</tabstop>
  <tabstop>note_as_</tabstop>
  <tabstop>note_a_</tabstop>
  <tabstop>note_ais_</tabstop>
  <tabstop>note_b_</tabstop>
  <tabstop>note_h_</tabstop>
  <tabstop>note_c__</tabstop>
  <tabstop>note_cis__</tabstop>
  <tabstop>note_des__</tabstop>
  <tabstop>note_d__</tabstop>
  <tabstop>note_dis__</tabstop>
  <tabstop>note_es__</tabstop>
  <tabstop>note_e__</tabstop>
  <tabstop>note_f__</tabstop>
  <tabstop>note_fis__</tabstop>
  <tabstop>note_ges__</tabstop>
  <tabstop>note_g__</tabstop>
  <tabstop>note_gis__</tabstop>
  <tabstop>note_as__</tabstop>
  <tabstop>note_a__</tabstop>
  <tabstop>note_ais__</tabstop>
  <tabstop>note_b__</tabstop>
  <tabstop>note_h__</tabstop>
  <tabstop>note_c___</tabstop>
  <tabstop>pushButton_start_training</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>action_Close</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
